packages/shared/src/events/generic-event-emitter.ts

```diff
@@ -1,21 +1,17 @@
-import EventEmitter from 'events';
```

Comment:
This import failed, and was the cause of the bug. Even though we bundle `events` in the `@openfeature/web-sdk`, it's not available here.